internal/v5/api.go

```diff
@@ -1010,22 +1010,15 @@ func (h *ReqHandler) putMetaPerm(id *router.ResolvedURL, path string, val *json.
 	if err := json.Unmarshal(*val, &perms); err != nil {
 		return errgo.Mask(err)
 	}
-	isPublic := false
```

Comment:
Removing this public field is a small step for a man, one giant leap for mankind.